FirebaseAppDistribution/Sources/FIRAppDistributionKeychainUtility.m

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#import <AppAuth/AppAuth.h>
+#import "FIRAppDistributionKeychainUtility+Private.h"
+
+NSString *const kFIRAppDistributionKeychainErrorDomain = @"com.firebase.app_distribution.keychain";
+
+@implementation FIRAppDistributionKeychainUtility
```

Comment:
FYI, there is a generic [keychain storage](https://github.com/firebase/firebase-ios-sdk/blob/e3f3a78aefaafdab4fcfb41d24ccc93b7a5279e9/GoogleUtilities/Environment/Public/GULKeychainStorage.h#L24) introduced recently in GoogleUtilities which may help you make the code simpler. Feel free to reach out to me for usage details.

Reply:
ooo! thank you for the tip! I will look into this - thanks!

Reply:
ooh this is awesome! Thanks @maksymmalyhin

Reply:
@maksymmalyhin would you be alright with merging this PR as is and following up in a subsequent change?

Reply:
Sure, it is definitely not a blocker to me. Just wanted to share the lib that may be useful for you.